tests/test_auth_dict.py

```diff
@@ -21,13 +21,15 @@ def tmp_home(tmpdir):
 
 
 def test_auth_from_dict():
-    """get_auth_dict, if given a dict, returns the input."""
+    """get_auth_dict, if given a dict, returns a shallow copy of the input."""
 
     some_dict = {'foo': 'bar'}
     result = get_auth_dict(some_dict)
 
-    # It should return exactly the input value.
-    assert result is some_dict
+    # Update as of Oct 2021
```

Comment:
I don't think this kind of comment is a good idea in general.  The info is always there in the git log if someone really wants to know what changed.

Usually if you're reading code, you want to know how the code works, and information about how the code *used to* work is irrelevant and just makes it harder to read.  As codebases get older, they'd become completely unreadable if people left comments there for every bug they fixed.